engine/src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -88,15 +89,16 @@ class WorkflowManagerActor(config: Config,
                            val workflowStore: ActorRef,
                            val serviceRegistryActor: ActorRef,
                            val workflowLogCopyRouter: ActorRef,
-                           val jobStoreActor: ActorRef,
-                           val dockerHashLookupActor: ActorRef)
+                           val jobStoreActor: ActorRef)
   extends LoggingFSM[WorkflowManagerState, WorkflowManagerData] {
 
   def this(workflowStore: ActorRef,
            serviceRegistryActor: ActorRef,
            workflowLogCopyRouter: ActorRef,
-           jobStoreActor: ActorRef,
-           dockerHashLookupActor: ActorRef) = this(ConfigFactory.load, workflowStore, serviceRegistryActor, workflowLogCopyRouter, jobStoreActor, dockerHashLookupActor)
+           jobStoreActor: ActorRef) = {
+    this(ConfigFactory.load, workflowStore, serviceRegistryActor, workflowLogCopyRouter, jobStoreActor)
```

Comment:
Is this more canonical scala-ese? I always thought `{ }` were overkill for one-liners